source/backend/apimodels/Models/Concepts/Lease/ConsultationLeaseMap.cs

```diff
@@ -10,18 +11,38 @@ public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<PimsLeaseConsultation, ConsultationLeaseModel>()
                 .Map(dest => dest.Id, src => src.LeaseConsultationId)
-                .Map(dest => dest.ConsultationType, src => src.ConsultationTypeCodeNavigation)
-                .Map(dest => dest.ConsultationStatusType, src => src.ConsultationStatusTypeCodeNavigation)
-                .Map(dest => dest.ParentLeaseId, src => src.LeaseId)
+                .Map(dest => dest.LeaseId, src => src.LeaseId)
+                .Map(dest => dest.Lease, src => src.Lease)
+                .Map(dest => dest.PersonId, src => src.PersonId)
+                .Map(dest => dest.Person, src => src.Person)
+                .Map(dest => dest.OrganizationId, src => src.OrganizationId)
+                .Map(dest => dest.Organization, src => src.Organization)
+                .Map(dest => dest.PrimaryContactId, src => src.PrimaryContactId)
+                .Map(dest => dest.PrimaryContact, src => src.PrimaryContact)
+                .Map(dest => dest.ConsultationTypeCode, src => src.ConsultationTypeCodeNavigation)
+                .Map(dest => dest.ConsultationStatusTypeCode, src => src.ConsultationStatusTypeCodeNavigation)
                 .Map(dest => dest.OtherDescription, src => src.OtherDescription)
+                .Map(dest => dest.RequestedOn, src => src.RequestedOn.ToNullableDateOnly())
+                .Map(dest => dest.IsResponseReceived, src => src.IsResponseReceived)
+                .Map(dest => dest.ResponseReceivedDate, src => src.ResponseReceivedDate.ToNullableDateOnly())
+                .Map(dest => dest.Comment, src => src.Comment)
                 .Inherits<IBaseAppEntity, BaseAuditModel>();
 
             config.NewConfig<ConsultationLeaseModel, PimsLeaseConsultation>()
                 .Map(dest => dest.LeaseConsultationId, src => src.Id)
-                .Map(dest => dest.LeaseId, src => src.ParentLeaseId)
-                .Map(dest => dest.ConsultationTypeCode, src => src.ConsultationType.Id)
-                .Map(dest => dest.ConsultationStatusTypeCode, src => src.ConsultationStatusType.Id)
+                .Map(dest => dest.LeaseId, src => src.LeaseId)
+                .Map(dest => dest.PersonId, src => src.PersonId)
+                .Map(dest => dest.OrganizationId, src => src.OrganizationId)
+                .Map(dest => dest.Organization, src => src.Organization)
+                .Map(dest => dest.PrimaryContactId, src => src.PrimaryContactId)
+                .Map(dest => dest.PrimaryContact, src => src.PrimaryContact)
+                .Map(dest => dest.ConsultationTypeCode, src => src.ConsultationTypeCode.Id)
+                .Map(dest => dest.ConsultationStatusTypeCode, src => src.ConsultationStatusTypeCode.Id)
                 .Map(dest => dest.OtherDescription, src => src.OtherDescription)
+                .Map(dest => dest.RequestedOn, src => src.RequestedOn.ToNullableDateTime())
```

Comment:
is this a DateOnly or a DateTime?

Reply:
Its a datetime, however we only need dates, that's wh y the model changes it